service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantAboutToStartControllerTest.java

```diff
@@ -45,17 +46,17 @@
 
     @BeforeEach
     void setup() {
-        given(organisationService.findOrganisation()).willReturn(POPULATED_ORGANISATION);
+        given(organisationService.findOrganisation()).willReturn(Optional.of(POPULATED_ORGANISATION));
```

Comment:
Given we are mocking this service do we need to mock the api as well? I think that one of these are a redundant mock.

Reply:
Well as this is integration tests it should mock only api and not service, thats for sure. But should not be in this commit, happy to create separate commit for this and unit test